utils/paramfetch/src/lib.rs

```diff
@@ -20,14 +22,16 @@ use tokio::{
     fs::{self, File},
     io::BufWriter,
 };
-use tokio_util::compat::FuturesAsyncReadCompatExt;
+use tokio_util::compat::TokioAsyncWriteCompatExt;
 
 const GATEWAY: &str = "https://proofs.filecoin.io/ipfs/";
 const PARAM_DIR: &str = "filecoin-proof-parameters";
 const DIR_ENV: &str = "FIL_PROOFS_PARAMETER_CACHE";
 const GATEWAY_ENV: &str = "IPFS_GATEWAY";
 const TRUST_PARAMS_ENV: &str = "TRUST_PARAMS";
 const DEFAULT_PARAMETERS: &str = include_str!("parameters.json");
+// Arbitrary limit at 10MB for DOS protection
+const MAX_PARAM_FILE_SIZE: usize = 10_000_000;
```

Comment:
One the files is ~280MB,  10MB might not be sufficient here

Reply:
Constant removed